libnavigation-ui/src/test/java/com/mapbox/navigation/ui/route/MapRouteLineTest.kt

```diff
@@ -1613,7 +1613,7 @@ class MapRouteLineTest {
             mapRouteLine.retrieveVisibility(),
             mapRouteLine.retrieveAlternativesVisible(),
             mapRouteSourceProvider,
-            0f,
+            0.0,
```

Comment:
Fixes conflict from https://github.com/mapbox/mapbox-navigation-android/pull/3661 and https://github.com/mapbox/mapbox-navigation-android/pull/3664

Reply:
:+1: